src/recordlinker/linking/old_matchers.py

```diff
@@ -51,7 +51,7 @@ def compare_strings(
         return rapidfuzz.distance.DamerauLevenshtein.normalized_similarity(string1, string2)
 
 
-def eval_perfect_match(feature_comparisons: list, **kwargs) -> bool:
```

Comment:
We don't use this module anymore, and I've been wondering if we should just delete it.  I created it a while back, thinking that we might want to port some of these over for future use, but we could always do that in the future just looking at revision history.  It might be worth just deleting now so others aren't confused in the future.

Reply:
Sounds good. I'll delete `old_matchers.py` and `test_old_matchers.py`.